### ISSUE TITLE
Add button to interrupt other uploads to a draft

### ISSUE THREAD

**Issue by nellh** [OpenNeuroOrg/openneuro#907]
If you have write access to a draft and an upload is in progress from another location, give the option to interrupt the other upload (take over the draft edit). Depends on #906.

@chrisfilo If you do interrupt it, this should probably notify the other location it was interrupted right? Also show who interrupted it?

**Comment by chrisgorgo** [comment]
Ideally, but I would make this feature a low priority.
